src/amber/cli/sentry_command_helper.cr

```diff
@@ -15,8 +15,8 @@ module Sentry
         name = Options.get_name
         {
           name:         name,
-          process_name: "./#{name}",
-          build:        "crystal build ./src/#{name}.cr",
+          process_name: "./bin/#{name}",
+          build:        "crystal build ./src/#{name}.cr -o bin/#{name}",
```

Comment:
I think we can use `"mkdir -p bin && crystal build ./src/#{name}.cr -o bin/#{name}"` to avoid error if we execute `amber watch` before `amber run`